examples/js/loaders/OBJLoader2.js

```diff
@@ -0,0 +1,996 @@
+/**
+  * @author Kai Salmen / https://kaisalmen.de
+  * Development repository: https://github.com/kaisalmen/WWOBJLoader
+  */
+
+'use strict';
+
+if ( THREE.OBJLoader2 === undefined ) { THREE.OBJLoader2 = {} }
+THREE.OBJLoader2.version = '1.0.1';
+
+/**
+ * Use this class to load OBJ data from files or to parse OBJ data from arraybuffer or text
+ * @class
+ *
+ * @param {THREE.DefaultLoadingManager} [manager] Extension of {@link THREE.DefaultLoadingManager}
+ */
+THREE.OBJLoader2 = (function () {
+
+	function OBJLoader2( manager ) {
+		this.manager = ( manager == null ) ? THREE.DefaultLoadingManager : manager;
+
+		this.path = '';
+		this.fileLoader = new THREE.FileLoader( this.manager );
+
+		this.meshCreator = new THREE.OBJLoader2.MeshCreator();
+		this.parser = new THREE.OBJLoader2.Parser( this.meshCreator );
```

Comment:
Builder function works! I pushed another update.
The minified versions of the bundles produce non-working worker code, but this is only a concern for the dev repo. I hope to find the error tomorrow.

Please, have a look at what I did, but don't merge it, yet as minified fix may introduce slight changes to builder functions.